src/store/modules/app/actions.js

```diff
@@ -115,7 +116,7 @@ const actions = {
       return p;
     }
     return axios
-      .get(backend_keyboards_url + '/' + state.keyboard)
+      .get(backend_keyboards_url + '/' + state.keyboard + '/info.json')
```

Comment:
we should probably replace this with a safer syntax
<pre>
`${backend_keyboards_url}/${state.keyboard}/info.json`
</pre>